test/irmin-pack/test_pack.ml

```diff
@@ -49,12 +49,20 @@ let stats = None
 
 let suite = { Irmin_test.name = "PACK"; init; clean; config; store; stats }
 
+let fresh_name =
+  let c = ref 0 in
+  fun () ->
+    incr c;
+    let name = Filename.concat test_dir ("pack_" ^ string_of_int !c) in
```

Comment:
This function should probably log an 'info' message or similar to indicate which resource it's created, otherwise it might be hard to find it after a test failure.

See: https://github.com/mirage/index/pull/84/files#diff-a51df4f9166627f8e5ce5f3951979f45R77